src/rttable.c

```diff
@@ -397,10 +399,12 @@ int insertRoute(uint32_t group, int ifx, uint32_t src) {
         // Register the VIF activity for the aging routine
         BIT_SET(croute->ageVifBits, ifx);
 
-        // Update dwnstrHosts
-        if(findDownstreamSlot(croute, src) == -1){
-            int free = findFreeDownstreamSlot(croute);
-            croute->dwnstrHosts[free] = src;
+        // Register dwnstrHosts for host tracking if fastleave is enabled
+	if(conf->fastUpstreamLeave) {
```

Comment:
Broken indentation (tab vs spaces), in igmpproxy are used consistently spaces.

Reply:
Missed that one, will fix.